face_track/face_track.py

```diff
@@ -376,15 +378,16 @@ def face_event_cb(self, data):
 	def face_loc_cb(self, data):
 		if not self.control_mode & self.C_FACE_TRACKING:
 			return
-		for face in data.faces:
-			fid = face.id
-			loc = face.point
-			# Sanity check.  Sometimes pi_vision sends us faces with
-			# location (0,0,0). Discard these.
-			if loc.x < 0.05:
-				continue
-			self.add_face(fid)
-
+		#below causes error in cmt .. mandeep
```

Comment:
what's the error? why is this a problem? 

If this code is stubbed out, then how does the robot know where to look?